config/config.go

```diff
@@ -35,16 +35,16 @@ func Get() (*Config, error) {
 		return cfg, nil
 	}
 
-	cfg := &Config{
+	cfg = &Config{
 		BindAddr:                   "localhost:27400",
 		AwsRegion:                  "eu-west-1",
 		DownloadBucketName:         "dp-interactives-file-uploads",
 		UploadBucketName:           "dp-interactives-vis-uploads",
 		Brokers:                    []string{"localhost:9092", "localhost:9093", "localhost:9094"},
 		KafkaVersion:               "1.0.2",
 		KafkaMaxBytes:              2000000,
-		InteractivesReadTopic:      "interactives-visualisations-api",
-		InteractivesWriteTopic:     "interactives-visualisations-importer",
+		InteractivesReadTopic:      "dp-interactives-read-topic",
+		InteractivesWriteTopic:     "dp-interactives-write-topic",
```

Comment:
Dont need a write topic as the reply is sent RESTfully

Reply:
will fix when we integrate dp-interactives-api client (still todo)